src/js/main.js

```diff
@@ -11,10 +11,11 @@ Site = {
     });
 
     $(document).ready(function () {
-      Site.Player.init();
+      //Site.Player.init();
```

Comment:
ah just for dev purposes. just so i didnt trigger the player every time i loaded the page